autorest/codegen/models/parameter_list.py

```diff
@@ -111,6 +111,8 @@ def is_parameter_in_signature(parameter):
                 or parameter.grouped_by
                 # If I'm body and it's flattened, I'm not either
                 or (parameter.location == ParameterLocation.Body and self.is_flattened)
+                # If I'm a kwarg, don't include in the signature
+                or parameter.is_kwarg
```

Comment:
Do you even need a parameter instance if it's a kwarg? Can't we just delete content_type at parsing time? Is there anything inside we need?